application/controllers/QuestionAdministrationController.php

```diff
@@ -1073,6 +1073,18 @@ public function actionImport()
         $iSurveyID = (int) App()->request->getPost('sid', 0);
         $gid = (int) App()->request->getPost('gid', 0);
 
+        if (!Permission::model()->hasSurveyPermission($surveyid, 'surveycontent', 'import')) {
+            App()->user->setFlash('error', gT("Access denied"));
+            $redirect = Yii::app()->createUrl(
```

Comment:
Bad redirect ? : goes to a second redirect.

Else : since it's a hacked POST : why not a 403 ?